services/QuillLMS/client/app/actions/district_standards_reports.js

```diff
@@ -31,12 +31,13 @@ export const initializePusherForDistrictStandardsReports = (adminId) => {
   }
 }
 
-export const getDistrictStandardsReports = () => {
+export const getDistrictStandardsReports = (isFreemiumView) => {
+  const freemiumParam = isFreemiumView ? '?freemium=true' : ''
   return (dispatch) => {
     requestGet(
-      `${process.env.DEFAULT_URL}/api/v1/progress_reports/district_standards_reports`,
+      `${process.env.DEFAULT_URL}/api/v1/progress_reports/district_standards_reports${freemiumParam}`,
       (body) => {
-        if (body.id) {
+        if (body.id && process.env.RAILS_ENV !== 'development') {
```

Comment:
What's the reasoning for excluding development here?  Should we also exclude test?

Reply:
There was just a lot of noise in the console while I was developing this so I turned it off, I should add test as well